superset/databases/ssh_tunnel/models.py

```diff
@@ -41,15 +42,17 @@ class SSHTunnel(Model, AuditMixinNullable, ExtraJSONMixin, ImportExportMixin):
     __tablename__ = "ssh_tunnels"
 
     id = sa.Column(sa.Integer, primary_key=True)
-    database_id = sa.Column(sa.Integer, sa.ForeignKey("dbs.id"), nullable=False)
+    database_id = sa.Column(
+        sa.Integer, sa.ForeignKey("dbs.id"), nullable=False, unique=True
+    )
     database: Database = relationship(
         "Database",
         backref=backref("ssh_tunnels", cascade="all, delete-orphan"),
         foreign_keys=[database_id],
     )
 
-    server_address = sa.Column(EncryptedType(sa.String, app_config["SECRET_KEY"]))
-    server_port = sa.Column(EncryptedType(sa.Integer, app_config["SECRET_KEY"]))
+    server_address = sa.Column(sa.Text)
```

Comment:
why aren't these encrypted any more?

Reply:
I was addressing this comment: https://github.com/apache/superset/pull/21912/files#r1035561067 which mentions we don't need those fields encrypted. Do we want to encrypt them? if yes I can rollback.

Reply:
They shouldn't need to be encrypted, IMO. Server hostname/port shouldn't contain sensitive info :)

Reply:
looks good.